altair/vegalite/v5/schema/mixins.py

```diff
@@ -1275,6 +1277,14 @@ def configure_title(self, *args, **kwargs) -> Self:
         copy.config["title"] = core.TitleConfig(*args, **kwargs)
         return copy
 
+    @use_signature(core.FormatConfig)
+    def configure_tooltipFormat(self, *args, **kwargs) -> Self:
```

Comment:
I'm not sure where it is in the codegen logic, but it would be nice to convert tooltipFormat to snake_case here (so it would be `configure_tooltip_format`).

Reply:
I'd also prefer snake_case but we would then also need to change some of the other `configure_` methods such as `configure_axisXTemporal`, `configure_headerColumn`, ...

Reply:
Ahh, ok, I didn't recall that we had others already in mixed case. Maybe we address this in Altair 6 some day 😃